### PR TITLE
Use `cidrsubnet` and `cidrhost` in scenarios and modules

### LINKED ISSUE #277: Get rid of `addr_3_octets`
For proper address and CIDR calculation, Terraform provides `cidrhost` and `cidrsubnet` functions

### PR DESCRIPTION
**Pull request by outcatcher**
Fix #277: Get rid of `addr_3_octets`

Add `network_cidr` variable to all scenarios, defaults to `192.168.0.0/16`

Example of subnet CIDR and host IP calculation:
```hcl
> cidrsubnet("192.168.0.0/16", 8, 5)
"192.168.5.0/24"
> cidrhost("192.168.5.0/24", 3)
"192.168.5.3"
```

### CODE REVIEW

**Review by lego963 — CHANGES_REQUESTED** [review]
then you have to change defavult values of `addr_3_octet` in `*.tfvars`

File: scenarios/csm_controller/terraform.tfvars [hunk 1]
```diff
@@ -2,3 +2,4 @@ availability_zone = "eu-de-01"
 scenario          = "csm_controller"
 ecs_flavor        = "s2.large.2"
 ecs_image         = "Debian_csm_test_host"
+network_cidr      = "192.168.0.0/16"
```
Comment (Polina-Gubina) [inline scenarios/csm_controller/terraform.tfvars:5]: network_cidr = "{{ terraform_network_cidr }}" ?

File: scenarios/modules/ecs_node/vm.tf [hunk 2]
```diff
@@ -80,7 +80,7 @@ resource "opentelekomcloud_networking_port_v2" "instance_port" {
   ]
   fixed_ip {
     subnet_id  = var.subnet_id
-    ip_address = "${var.net_address}.${count.index + 10}"
+    ip_address = var.node_local_ip
```
Comment (anton-sidelnikov) [inline scenarios/modules/ecs_node/vm.tf]: how it works with multiple nodes?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: scenarios/lb_monitoring/infrastructure.tf [hunk 3]
```diff
@@ -15,30 +15,31 @@ module "nodes" {
...
+
 module "loadbalancer" {
   source = "../modules/loadbalancer"
 
   instances        = module.nodes.instances
-  net_address      = var.addr_3
   scenario         = var.scenario
   subnet_id        = var.subnet_id
   workspace_prefix = local.workspace_prefix
+  lb_local_ip      = cidrhost(local.scenario_subnet, 1)
```
Comment (anton-sidelnikov) [inline scenarios/lb_monitoring/infrastructure.tf]: cidrhost(local.scenario_subnet, 3)

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by Polina-Gubina — DISMISSED** [review]

**Comment by outcatcher** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

**Comment by outcatcher** [comment]
recheck

**Review by Polina-Gubina — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Pull request merged**